src/cbapi/psc/livequery/models/device_summary.yaml

```diff
@@ -0,0 +1,37 @@
+type: object
+required: [] # TODO(ww): Find out which result fields are required
+properties:
+  id:
+    type: string
+    description: The result's unique ID
+  total_results:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: Number of results returned for this particular device
+  device_id:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: The device's unique ID
+  device_name:
+    type: string
+    description: The device's name
+  time_received:
+    type: string
+    description: The time at which this result was received # NOTE(ww): This is a guess
+  status:
+    type: string
+    description: The result's status
+  device_message:
+    type: string
+    description: Placeholder # TODO(ww): Needs description
+  policy_id:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: The device's policy ID
+  policy_name:
+    type: string
+    description: The device's policy name
+  metrics: # TODO(ww): Document each field in metrics?
+    type: object
+    description: Metrics associated with the device
+    
```

Comment:
Looks like we are missing `OS` as a property

Reply:
https://developer.carbonblack.com/reference/psc/cb-liveops/latest/livequery-api/#device-view-filters

Reply:
Good catch.